src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -1906,13 +1906,15 @@ public enum ClassOption {
                 this.flag = flag;
             }
 
-            static int optionsToFlag(Set<ClassOption> options) {
+            static int optionsToFlag(ClassOption[] options) {
                 int flags = 0;
                 for (ClassOption cp : options) {
                     flags |= cp.flag;
```

Comment:
If we decide to throw IAE for duplicate flags, then we might have something like:
```java
if ((flags & cp.flag) != 0)
    throw new IllegalArgumentException(); // Maybe a custom message
```
before this line.